src/objects/npe01__Contacts_And_Orgs_Settings__c.object

```diff
@@ -95,6 +95,18 @@
         <type>Number</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Multiple_Tributes__c</fullName>
+        <description>Allows multiple Tributes per Opportunity.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>Valid values: 1) &quot;Allow only one tribute per Opportunity record&quot; 2) “Allow only one tribute per Opportunity, or multiple tributes on Tribute records” 3) “Allow only multiple Tribute records and migrate Tribute data entered on Opportunities to Tributes&quot;</inlineHelpText>
+        <label>Multiple Tributes</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
```

Comment:
Should this field be a simple Integer with either 1, 2, or 3? My concern about storing the entire picklist string value is that it might break if the User translates the picklist values (which will eventually be done as part of the normal package). It's worth testing this to make sure everything works with the labels are translated and you have Users in multiple languages.

Reply:
@juliesnow - What is your take on this one?  Should be change this?

Reply:
We have found bugs with storing picklist and record type labels in custom settings when it comes to translations. Sticking with a simple integer will avoid that and allow the product to work even if someone changes the label content (which anyone can do in any language). I'm **very** highly recommending this change.

Reply:
Also, this field will default to `null` for all existing customers. I didn't notice a _default_ value/interpretation for `null` in the code, though I might have missed it.

Reply:
null to default is great callout.  Am wrapping that and returning the Single value now.  I will speak with the team about the 1,2,3 versus Strings-- totally get what you are saying...and will likely incorporate.  Will respond back if there is significant pushback with why.

Reply:
Yeah, the big place this breaks down is in multilingual orgs, correct?  This is because the stored value will only be of one language, but the labels will rendered in the user's language, if I am understanding this correctly?  If that is the case, I definitely agree this has to be changed.

Reply:
@force2b  - In the inline help text, there's a combination of both encoded quotes and keyed quotes (&quot; and "). Is that an issue at all? What's our convention for special characters anyway?

Reply:
Great catch @sam-knox. These should all be encoded for consistency, but it's not earth-shattering if they're missed. It seems that the build and packaging process is able to work with these without failing. Still we should follow best practices on encoding.

Reply:
@sam-knox, @force2b - Can you indicate which file/field this is?  I could not find the mixed mode stuff, only the encoded ones.

Reply:
@force2b - Please re-review.  With the exception of breaking out the tests from the OPP_OpportunityContactRoles_TEST class, the internationalization of the new object, and the encoding issue, the other requested changes are in.

Reply:
@jbrandolinisf - The place I saw the encoding issue was on line 102 in npe01__Contacts_And_Orgs_Settings__c.object.

Reply:
Confirmed the encoding issue was from a prior version, and is no longer present with @sam-knox.